src/app/styles/overrides.scss

```diff
@@ -200,4 +200,9 @@ accordion-group + accordion-group {
 .edit--icon {
   cursor: pointer;
   float: right;
+}
+
+.deam-container {
+  padding-left: 7rem!important;
+  padding-right: 7rem!important;
```

Comment:
I think there needs to be a space between the measurement and the `!important` to be compatible with all browsers:
```
padding-left: 7rem !important;
padding-right: 7rem !important;
```

Reply:
Thanks Harry. It's fixed now in my latest push.